pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraLocationOfOriginalTest.scala

```diff
@@ -0,0 +1,58 @@
+package uk.ac.wellcome.platform.transformer.sierra.transformers
+
+import org.scalatest.{FunSpec, Matchers}
+import uk.ac.wellcome.platform.transformer.sierra.generators.{
+  MarcGenerators,
+  SierraDataGenerators
+}
+import uk.ac.wellcome.platform.transformer.sierra.source.{
+  MarcSubfield,
+  SierraBibData,
+  VarField
+}
+
+class SierraLocationOfOriginalTest
+    extends FunSpec
+    with Matchers
+    with MarcGenerators
+    with SierraDataGenerators {
+
+  val content = "cite as content"
```

Comment:
Might make testing confusing with this content?

Reply:
oops too much copy / paste

Reply:
Was thinking if you could have a base test class for this where you just give it some values to run against, but copy paste in testing is normally fine.

Reply:
True, these tests are essentially the same and lots of this could be abstracted away. Gonna leave as is for now as just test cases, but will have a look at some point (probs other transformer tests fit this pattern too).